storage/innobase/xtrabackup/doc/source/advanced/page_tracking.rst

```diff
@@ -0,0 +1,126 @@
+.. _page_tracking:
+
+====================================
+Page Tracking for Incremental Backup 
+====================================
+
+This feature is *tech preview* quality.
+
+Percona XtraBackup 8.0.27 adds support for the page tracking functionality for `incremental backup <https://www.percona.com/doc/percona-xtrabackup/8.0/backup_scenarios/incremental_backup.html>`__ . 
+
+To create an incremental backup with page tracking, Percona XtraBackup (PXB) uses the MySQL mysqlbackup component. This component looks in the InnoDB data files for pages that have been modified since the last backup and copies only those changed pages. This operation removes the need to scan all of the pages in the database. If the number of changed pages is not over 50%, the page tracking feature can improve the speed of incremental backups.
+
+Install the component
+---------------------
+
+To start using the page tracking functionality, do the following steps:
+
+1. Install the mysqlbackup component and enable it on the server: 
+
+  .. code-block:: bash
+
+     INSTALL COMPONENT "file://component_mysqlbackup";
+
+2. Check whether the mysqlbackup component is installed succesfuly:
+
+  .. code-block:: bash
+
+     SELECT COUNT(1) FROM mysql.component WHERE component_urn='file://component_mysqlbackup';
+
+.. note::
+
+   The mysqlbackup component processes all the tracked pages in memory. If the component does not have enough available memory to process all the tracked pages, the component throws an error and exits. For example, if an incremental backup uses 200GB, the component can additionally use about 100MB of memory to store the page tracking data.
```

Comment:
replace mysql backup component with PXB.  Also move this section to the end.

Reply:
The section should stay before the examples. The user should know this information before they use the option.

Reply:
Updated